index.html

```diff
@@ -199,6 +199,14 @@
 
 ### What points of centralization does your feature bring to the web platform? {#centralization}
 
+Centralization can occur when a feature gives disproportionate influence to a
+few entities, whether through control over information flows, trust decisions,
+or other critical functions. Centralization can reduce openness, limit
+individual choice, and create risks of exclusion, manipulation, or abuse.
+Consider how a centralised design can affect governance, technical operations,
+and the broader ecosystem, and how these effects interact with the factors
+that drive centralization.
+
```

Comment:
I think lines 206 - 208 should come after the section that's already there (lines 210 onwards). Prompting readers with questions about centralization then concluding with what they should consider.

Reply:
Sounds good. Moved https://github.com/w3ctag/societal-impact-questionnaire/pull/23/commits/fcabaf071ca73ac80dec873ef87ff84c5f62480e